fairlearn/post_processing/threshold_optimizer.py

```diff
@@ -429,7 +429,7 @@ def _reformat_data_into_dict(key, data_dict, additional_data):
                                               additional_data.shape[1] > 1):
             # TODO: extend to multiple columns for additional_group data
             raise ValueError(
-                MULTIPLE_DATA_COLUMNS_ERROR_MESSAGE.format("group_data"))
+                MULTIPLE_DATA_COLUMNS_ERROR_MESSAGE.format("sensitive_features"))
```

Comment:
`_KW_GROUP_MEMBERSHIP` ? And on line 446 below.

Reply:
again... not sure.

Reply:
If they aren't kwargs, leave them. I'd lost track of which file I was looking at.